adele-clerk/src/main/java/com/microservicesteam/adele/clerk/boundary/web/ReservationController.java

```diff
@@ -0,0 +1,25 @@
+package com.microservicesteam.adele.clerk.boundary.web;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.microservicesteam.adele.clerk.domain.ReservationRequest;
+import com.microservicesteam.adele.clerk.domain.ReservationResponse;
+import com.microservicesteam.adele.clerk.domain.ReservationsService;
+import lombok.AllArgsConstructor;
+
+@RestController
+@RequestMapping("reservations")
+@AllArgsConstructor
+public class ReservationController {
+
+    private final ReservationsService reservationsService;
+
+    @PostMapping
+    public ReservationResponse reservePositions(@RequestBody ReservationRequest reservationRequest) {
```

Comment:
We should rename `position` to `ticketId`.
We should rename `reservePositions` to `reserveTickets`.
And also `ReservationRequest` should contains List<TicketId>.